src/components/Facility/BedCapacity.tsx

```
@@ -117,64 +110,62 @@ export const BedCapacity = (props: BedCapacityProps) => {
-        errors[field] = "Total capacity cannot be negative";
-        invalidForm = true;
+        validForm = false;
+      } else if (field === "currentOccupancy" && Number(form[field] < 0)) {
+        if (Number(form[field] < 0)) {
+          errors[field] = "Occupied cannot be negative";
+          validForm = false;
+        } else if (Number(form[field]) > Number(form.totalCapacity)) {
+          errors[field] =
+            "Occupied must be less than or equal to total capacity";
+          validForm = false;
+        } else if (isNaN(Number(form[field]))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)